tools/cldr-code/src/main/java/org/unicode/cldr/util/CLDRPaths.java

```diff
@@ -111,25 +128,36 @@
 
     /**
      * @deprecated Don't use this from any code that is run from the .JAR (SurveyTool, tests, etc).
-     *             If it must be used, add a comment next to the usage to explain why it is needed.
+     *     If it must be used, add a comment next to the usage to explain why it is needed.
      */
-    @Deprecated
-    public static final String UTIL_DATA_DIR = FileReaders.getRelativeFileName(CldrUtility.class, "data/");
+    @Deprecated public static final String UTIL_DATA_DIR = FileReaders.getRelativeFileName(CldrUtility.class, "data/");
 
     public enum DIRECTORIES {
-        common_dtd, common_properties, common_uca,
+        common_dtd,
```

Comment:
Seems like a big win for enum formatting, though this case is clearly busted in the current code.